nikola/nikola.py

```diff
@@ -2581,6 +2585,13 @@ def generic_index_renderer(self, lang, posts, indexes_title, template_name, cont
                     context['current_page'] = ipages_i - 1
                 context['prev_next_links_reversed'] = kw['indexes_static']
             context["permalink"] = page_links[i]
+            context["is_frontmost_index"] = i == 0
+
+            # Add dependencies to featured posts
+            if 'featured' in context:
```

Comment:
I think we should mention that in the docstring of `generic_index_renderer`, in case someone wants to put something else called `featured` in there.